instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -1,11 +1,25 @@
 ---
 rules:
+
+  # Context Level Metrics
+
   - bean: org.apache.camel:context=*,type=context,name=*
     prefix: camel.context.
     metricAttribute:
       context: param(context)
       camelVersion: beanattr(CamelVersion)
+      packageScanClassResolver: beanattr(PackageScanClassResolver)
     mapping:
+      StartedRoutes:
+        metric: route.started
+        type: updowncounter
+        unit: "{routes}"
```

Comment:
The semantic conventions for metric units recommend using singular form, so {route} rather than {routes}.